keyboards/omnikeyish/omnikeyish.c

```diff
@@ -0,0 +1,64 @@
+#include "matrix.h"
```

Comment:
All of these includes should be removed, and replaced with just: 

```suggestion
#include "omnikeyish.h"
```